store/sqlstore/upgrade.go

```diff
@@ -531,6 +531,10 @@ func UpgradeDatabaseToVersion56(sqlStore SqlStore) {
 			sqlStore.RemoveIndexIfExists("idx_users_firstname_lower", "lower(FirstName)")
 			sqlStore.RemoveIndexIfExists("idx_users_lastname_lower", "lower(LastName)")
 		}
+
+		sqlStore.CreateIndexIfNotExists("idx_groupmembers_create_at", "GroupMembers", "CreateAt")
+		sqlStore.CreateIndexIfNotExists("idx_groups_remote_id", "Groups", "RemoteId")
```

Comment:
Before we merge to master, this will need to go in the update to 5.8 schema method.